core/templates/css/oppia.css

```diff
@@ -1943,6 +1943,7 @@ pre.oppia-pre-wrapped-text {
   display: block;
   margin-left: auto;
   margin-right: auto;
+  object-fit: contain;
```

Comment:
Note that this applied across the app where the "center-element-horizontally" class is used and not just blog page. Please add screenshots of the other scenarios e.g. check the image click input interaction and images in the exploration editor work as expected.

Reply:
Hey @kevintab95 PTAL
https://github.com/user-attachments/assets/e7d128c3-4a99-4553-9eb4-add6e0515593